cardano-wallet.cabal

```diff
@@ -111,6 +111,7 @@ test-suite unit
     , bytestring
     , cardano-crypto
     , cardano-wallet
+    , cassava
```

Comment:
If anything, I'd suggest to use `JSON` as a (semi-)structured data format for storing data. We can cut-out an extra dependency to a csv parser this way :+1:

Reply:
done